test/test_ident.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+
+from fire.api.model import (
+    PunktInformation,
+    PunktInformationType,
+    PunktInformationTypeAnvendelse,
+    Ident,
+)
+
+
+def lav_ident(variant, ident):
+    pit = PunktInformationType(
+        name=variant,
+        anvendelse=PunktInformationTypeAnvendelse.TEKST,
+        beskrivelse="Bare en test",
+    )
+    return Ident(PunktInformation(infotype=pit, tekst=ident))
+
+
+def test_ident_sortering():
+    gi = lav_ident("IDENT:GI", "G.M.902")
+    gnss = lav_ident("IDENT:GNSS", "SKEJ")
+    gnss2 = lav_ident("IDENT:GNSS", "AAAA")
+    landsnr = lav_ident("IDENT:landsnr", "102-08-00802")
+    jessen = lav_ident("IDENT:jessen", "81412")
+    station = lav_ident("IDENT:station", "1321")
+    ekstern = lav_ident("IDENT:ekstern", "88-01-2342")
+    diverse = lav_ident("IDENT:diverse", "flaf")
+    refgeoid = lav_ident("IDENT:refgeo_id", "12345")
```

Comment:
Så alle de numeriske ident-typer er også gemt som tekst i FIRE? Jeg har ikke rigtigt noget overblik over hvad der sker i migreringen desangående, men [her](https://github.com/Kortforsyningen/FIRE/blob/7b8cc6a0028d24951fc8b014429b1774eee4e85a/sql/ddl.sql#L135-L155) definerer vi jo både en tekst og en taltype og en indikator til at skelne mellem hvad der gælder. Bruger vi reelt "tal" til noget (ja - det er et spørgsmål lidt uden for reviewobjektet, men det ville da være fint at smide ubrugt kompleksitet ud)

Reply:
> Så alle de numeriske ident-typer er også gemt som tekst i FIRE?

Ja, se kommentar ovenfor.

> Bruger vi reelt "tal" til noget (ja - det er et spørgsmål lidt uden for reviewobjektet, men det ville da være fint at smide ubrugt kompleksitet ud)

Ja, det gør vi, fx `ATTR:højde_over_terræn`.

Reply:
Tak!